### DISCUSSION TITLE
NamedCsvReader

### DISCUSSION THREAD

**Discussion by illdd1** [osiegmar/FastCSV#78] [Q&A] [answered]
I am using NamedCsvReader and it is giving errors when there is an empty row or and if the number of rows do not match the amount of rows in the header. It is possible to avoid errors in CsvReader but not NamedCsvReader is there any way to avoid this issue?

**Comment by osiegmar** [comment]
Rows with different field count is currently not supported by NamedCsvReader. I didn't implement it as I considered it pretty error prone. Empty rows (rows only containing end-of-line character(s)) should be supported, though.

**Comment by osiegmar** [comment, marked as answer]
Will be implemented by #87
